packages/core/contracts/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.sol

```diff
@@ -38,6 +38,7 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
         FixedPoint.Unsigned minSponsorTokens;
         uint256 withdrawalLiveness;
         uint256 liquidationLiveness;
+        address beneficiary;
```

Comment:
nit: beneficiary as a variable name makes sense in the context of this PR but taken out of context it is a bit confusing. perhaps a different variable name for this?

Reply:
prehaps `excessTokenBeneficiary`?

Reply:
SGTM!

Reply:
Done.